scripts/incentive-spreadsheet-registry.ts

```diff
@@ -44,6 +44,16 @@ export const FILES: { [ident: string]: IncentiveFile } = {
     enHeader: 'Program Description (guideline)',
     esHeader: 'Program Description (Spanish)',
   },
+  DC: {
+    filepath: 'data/DC/incentives.json',
+    sheetUrl:
+      'https://docs.google.com/spreadsheets/d/e/2PACX-1vQb8t0MXV32iNBujzP4s5saPZl7qpaBcVckDCqSwzHHKAQVdaQtBvE2FgPCv1XvwDGN5ORAlXtKMqmp/pub?gid=995688950&single=true&output=csv',
+    headerRowNumber: 2,
+    idHeader: 'ID',
```

Comment:
I just merged a PR getting rid of some of these fields, so you'll need to delete them when you merge.